python/pyspark/sql/connect/client.py

```diff
@@ -125,13 +126,30 @@ def metadata(self) -> typing.Iterable[typing.Tuple[str, str]]:
 
     @property
     def secure(self) -> bool:
+        if self._token is not None:
+            return True
+
         value = self.params.get(ChannelBuilder.PARAM_USE_SSL, "")
         return value.lower() == "true"
 
     @property
     def endpoint(self) -> str:
         return f"{self.host}:{self.port}"
 
+    @property
+    def _token(self) -> Optional[str]:
+        return self.params.get(ChannelBuilder.PARAM_TOKEN, None)
+
+    @property
+    def user_id(self) -> Optional[str]:
```

Comment:
should probably follow camelCase ..  Are these a lot to fix? Wonder if we can fix them here together if there aren't.

Reply:
I can fix this here (still struggling to get used to camelCase method names) :)

Reply:
I changed it please have a look.